ceph.spec.in

```diff
@@ -504,6 +505,18 @@ Requires:       ceph-mgr = %{_epoch_prefix}%{version}-%{release}
 ceph-mgr-rook is a ceph-mgr plugin for orchestration functions using
 a Rook backend.
 
+%package mgr-ssh
+Summary:        ceph-mgr ssh plugin
```

Comment:
might want to use module instead of plugin. i recall that @LenzGr mentioned that they are actually modules instead of plugins over IRC. we need to at least update the user facing text first. i also found that all option names related to mgr modules are using "modules" in their names. for instance, "mgr_module_path" and "mgr_initial_modules".

Reply:
fixed.